internal/serve/validators/wallet_validator.go

```diff
@@ -42,6 +44,13 @@ func (wv *WalletValidator) ValidateCreateWalletRequest(ctx context.Context, reqB
 
 	wv.Check(name != "", "name", "name is required")
 	wv.Check(homepage != "", "homepage", "homepage is required")
+	if homepage != "" && enforceHTTPS {
+		schemes := []string{"https"}
+		if !enforceHTTPS {
```

Comment:
I believe this is always false, you're already checking for `enforceHTTPS` on line 47.